DataAccessGeneration/Settings.cs

```diff
@@ -10,7 +10,8 @@ public class Settings
         public List<ProcedureSetting> ProcedureList { get; set; } = new List<ProcedureSetting>();
         public bool IncludeFakes { get; set; } = true;
         public string? RepositoryName { get; set; } = null;
-        
+        public bool AllowExecution { get; set; } = true;
```

Comment:
probably want to default to false, since nothing before was being executed

Reply:
Prior to this version, it was being executed

Reply:
This gives the option to turn it off.